samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -320,7 +323,11 @@ static void mix_mono_to_stereo(enum bt_audio_location channels)
 		memset(&((uint8_t *)usb_audio_data)[size], 0, sizeof(usb_audio_data) - size);
 	}
 
+#if defined(CONFIG_TARGET_BROADCAST_CHANNEL)
 	cidx = get_channel_index(channels, CONFIG_TARGET_BROADCAST_CHANNEL);
+#else
+	cidx = 0;
+#endif /* CONFIG_TARGET_BROADCAST_CHANNEL */
```

Comment:
```suggestion
	if (IS_ENABLED(USE_SPECIFIC_BROADCAST_CHANNEL) {
		cidx = get_channel_index(channels, CONFIG_TARGET_BROADCAST_CHANNEL);
	} else {
		cidx = 0;
	}
```

Reply:
Yes. Will use suggested code

Reply:
Actully I cannot because CONFIG_TARGET_BROADCAST_CHANNEL does not exist when USE_SPECIFIC_BROADCAST_CHANNEL is not enabled